scripts/globals/items/beitetsu_parcel.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- ID: 6181
+-- Beitetsu Parcel
+-- Breaks up a Beitetsu Parcel
+-----------------------------------
+require("scripts/globals/msg")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:getFreeSlotsCount() == 0 then
+        result = xi.msg.basic.ITEM_NO_USE_INVENTORY
+    end
+    return result
+end
+
+item_object.onItemUse = function(target)
+    target:addItem(4060, math.random(3, 15))
+end
+
+return item_object
+
```

Comment:
From CI:
```
scripts/globals/items/beitetsu_parcel.lua:23: Found multiple newline characters next to each other, please replace these with single newlines.
```

Reply:
You had it correct in this commit: https://github.com/LandSandBoat/server/pull/2677/commits/f29ac19fcaebbde2066f3cf7f5ab262e15fe0c0c